Gulpfile.coffee

```diff
@@ -205,32 +112,24 @@ gulp.task '_watchLint', ['lint'], ->
   gulp.watch '*.coffee', ['lint']
 
 
-gulp.task '_webserver', ->
-  livereload.listen()
-  config =
-    port: process.env['PORT'] or 8000
-    # host: '0.0.0.0'
-    # livereload: true # Use the livereload.listen to notify when
-    # the CSS file is rebuilt
-    fallback: 'index.html'
-    middleware: -> [
-      cors(), # For font loading from tutor-server
-      (req, res, next) ->
-        if req.url.match(/\.svg$/)
-          res.setHeader('Content-Type', 'image/svg+xml')
-        if req.url.match(/woff[2]?$/)
-          name = req.url[req.url.lastIndexOf('/') + 1 .. req.url.length]
-          ext  = name[name.lastIndexOf('.') + 1 .. name.length]
-          res.setHeader("Content-Disposition", "attachment; filename=\"#{name}\"")
-          res.setHeader('Content-Type', "application/font-#{ext}")
-        next()
+gulp.task '_karma', ->
+  server = new karma.Server(KARMA_CONFIG)
+  server.start()
```

Comment:
I think it'll probably make sense to have two karma tasks.  One with coverage report and one for `tdd` that won't run it every single time a file's modified.